tests/spread-function-call/spread-function-call.es5

```diff
@@ -0,0 +1,47 @@
+var list = [1, 2, 3];
+
+Array.prototype.expand = false;
+
+function __(list) {
+  list.expand = true;
+  return list;
+}
+
+// wrap the function to be called, so that the arguments are parsed first.
+function wrap(func) {
```

Comment:
Thanks for the submission. I have some concerns about wether anyone would write this or not in a non-ES6 environment. Were you using a pattern like this in your own code prior to ES6 becoming common place?